### ISSUE TITLE
Create an initial wireframe for 'Cultural Organisations in Leeds'

### ISSUE THREAD

**Issue by taz-chiles** [open-innovations/leeds-schools-cultural-engagement#39]
A new page to be created - 'Cultural Organisations in Leeds' 

- [ ] Explore data and initial wireframes to see what can be visualised on the page
- [ ] Create an initial mockup to share with Briony when I speak with her this week. 

We may wish to show:

Headline stats:

- Number of cultural organisations funded by LCC
- Number of NPOs in Leeds
- Number of Arts Award centres in Leeds
- Number of NPOs which are Arts Award centres.
- Might we also want to show the number of Arts Council funded projects?

Map:

3 different coloured markers, highlighting:

- NPOs
- LCC funded organisations
- Other organisations that were mentioned in the survey.

Tooltip to include (lower priority):

- Does this organisation offer an Arts Award? Y/N?
- Link to their website

Other potential visualisations:

- A table to rank NPOs by number of engagements with schools
- A table to rank NPOs by amount of funding from ACE
- A table to rank organisations by funding from LCC
- A table or headline figure for Music Education Partnerships (more data on this to follow from LCC)

Unblocks #20
